hack/crds-utils.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+# Copyright 2020 Red Hat, Inc. and/or its affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Script with utils for CRDs resources.
+
+CRD_FILES_TO_IMPORT=(
+  "app.kiegroup.org_kogitoapps_crd.yaml"
+  "app.kiegroup.org_kogitodataindices_crd.yaml"
+  "app.kiegroup.org_kogitoinfras_crd.yaml"
+  "app.kiegroup.org_kogitojobsservices_crd.yaml"
+  "app.kiegroup.org_kogitomgmtconsoles_crd.yaml"
+  )
+
+# Apply CRDs by replacing existing resources or applying if they didn't exist previously.
+#   Example: apply_crds my_path
+function apply_crds(){
+  deploy_folder=${1}
+  for file in ${CRD_FILES_TO_IMPORT[*]}
```

Comment:
Why not using this method instead?
https://github.com/kiegroup/kogito-cloud-operator/blob/master/hack/scorecard.sh#L17

This way we don't have to keep updating those files each time we have a new CRD.

Reply:
Thanks for this comment! 
I've already updated the script so it can inspect the existing files for applying and downloading. This way, we don't need to update this file if we delete/add new CRDs file.